src/qml/components/PeersIndicator.qml

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The PeersIndicator component.
```

Comment:
I guess there's no need for this line, just look at the path and name of the file